cli/src/potpie/main.py

```
@@ -0,0 +1,340 @@
+def conversation():
+    """Talk with your conversations"""
+    pass
+
+
+@conversation.command()
+@click.argument("title")
+def create(title):
+    """Create a new conversation"""
+    base_url: str = Utility.base_url()
+    # Sees that user_id is used as the defaultUsername
+    user_id = os.getenv("defaultUsername", "defaultuser")
```

Ruff (0.8.2)

Use capitalized environment variable `DEFAULTUSERNAME` instead of `defaultUsername`

(SIM112)